### PR TITLE
nodejs-stats-webpack-plugin

### PR DESCRIPTION
**Pull request by dLobatog**

### CODE REVIEW

File: nodejs-stats-webpack-plugin/nodejs-stats-webpack-plugin.spec [hunk 1]
```diff
@@ -0,0 +1,52 @@
+%global npm_name stats-webpack-plugin
+
+Name: nodejs-%{npm_name}
+Version: 0.2.2
+Release: 1%{?dist}
+Summary: Write the stats of a build to a file
+License: undefined
```
Comment (domcleal) [inline nodejs-stats-webpack-plugin/nodejs-stats-webpack-plugin.spec]: MIT

File: nodejs-stats-webpack-plugin/nodejs-stats-webpack-plugin.spec [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+Group: Development/Libraries
+URL: undefined
```
Comment (domcleal) [inline nodejs-stats-webpack-plugin/nodejs-stats-webpack-plugin.spec]: https://github.com/unindented/stats-webpack-plugin

File: nodejs-stats-webpack-plugin/nodejs-stats-webpack-plugin.spec [hunk 3]
```diff
@@ -0,0 +1,52 @@
+%global npm_name stats-webpack-plugin
+
+Name: nodejs-%{npm_name}
+Version: 0.2.2
```
Comment (domcleal) [inline nodejs-stats-webpack-plugin/nodejs-stats-webpack-plugin.spec]: Ideally up to 0.4.0, the latest (`^0.2.1`).

**Comment by dLobatog** [comment]
Updated - this PR also contains an update to comps_doc to handle the nodejs packages

**Comment by dLobatog** [comment]
Ah my bad, it failed because it still has the old cache. I'll update that

**Comment by dLobatog** [comment]
Looks like the author didn't pack README.md in the tar... http://koji.katello.org/koji/getfile?taskID=508648&name=build.log - I've changed the %doc section now, it's an empty -doc package but in case in the future the tar includes docs, we can put them there.

**Comment by domcleal** [comment]
Merged as 6a77248, thanks @dLobatog.

**Pull request closed without merging**